src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -59,6 +64,18 @@ export const AssetActions: React.FC<FoxTabProps> = ({ assetId }) => {
   const handleReceiveClick = () =>
     isConnected ? receive.open({ asset, accountId }) : handleWalletModalOpen()
 
+  useEffect(() => {
+    if (!accountId) return
+    const { account: foxUserAddress } = fromAccountId(accountId)
+    const coinbasePayFoxLink = generateOnRampURL({
+      appId: getConfig().REACT_APP_COINBASE_PAY_APP_ID,
+      destinationWallets: [{ address: foxUserAddress, assets: ['FOX'] }],
+    })
+    setCoinbasePayLink(coinbasePayFoxLink)
+  }, [accountId])
+
+  const coinbasePayFeatureFlag = getConfig().REACT_APP_FEATURE_COINBASE_RAMP
```

Comment:
Can we declare this outside of the component? 🙏🏽

Reply:
Done.